libs/@guardian/libs/src/logger/@types/logger.ts

```diff
@@ -1,7 +1,7 @@
-import type { commonStyle, teamStyles } from '../teamStyles';
+import type { commonStyle, subscriptionStyles } from '../subscriptions';
 
-export type Teams<K extends string> = Record<K, Record<string, string>>;
-export type LogCall = (team: TeamName, ...args: unknown[]) => void;
-export type TeamSubscription = (arg: TeamName) => void;
-export type TeamStyle = TeamName | keyof typeof commonStyle;
-export type TeamName = keyof typeof teamStyles;
+export type Subscriptions<K extends string> = Record<K, Record<string, string>>;
+export type LogCall = (subscription: Subscription, ...args: unknown[]) => void;
+export type SubscriptionStyle = (arg: Subscription) => void;
```

Comment:
I’m not sure this is aptly named… is this the method that allows you to subscribe and unsubscribe?

```suggestion
export type Subscribe = (arg: Subscription) => void;
```

Reply:
oh yeah looks liek it - it's really hard to know with the types all on their own and now descriptions. it's a confusing pattern